backend/danswer/one_shot_answer/answer_question.py

```diff
@@ -204,12 +213,17 @@ def stream_answer_objects(
         # tested quotes with tool calling too much yet
         skip_explicit_tool_calling=True,
         return_contexts=query_req.return_contexts,
+        skip_generation=True,
     )
-    # won't be any ImageGenerationDisplay responses since that tool is never passed in
+
+    # won't be any ImageGenerationDisplay responses since that tool is never passed in go
```

Comment:
go?

Reply:
Keyboard shortcuts betraying me (likely meant to use command bar space)